tests/ui/app_list.js

```diff
@@ -393,10 +392,11 @@ casper.test.begin('Test collection detail page for app tile expanded state.', {
         });
 
         // Visit a collection details page and check it's not expanded.
-        helpers.startCasper({path: '/feed/collection/top-games'});
-        helpers.waitForPageLoaded(function() {
-            test.assertDoesntExist('.app-list.expanded');
-            test.assertDoesntExist('.previews');
+        casper.thenOpen(helpers.makeUrl('/feed/collection/top-games'), function() {
+            helpers.waitForPageLoaded(function() {
+                test.assertDoesntExist('.app-list.expanded');
```

Comment:
I left this test in as it might be checking for a regression. Should I remove it?

/cc @spasovski